lib/core/modules/modules.js

```diff
@@ -132,3 +132,7 @@ export function getByCategory(category: string): Array<Module<any>> {
 			return (i18n(a.moduleName).toLowerCase() > i18n(b.moduleName).toLowerCase()) ? 1 : -1;
 		});
 }
+
+export function getOptionValue(opaqueId: OpaqueModuleId, option: string) {
```

Comment:
I could have sworn there was already some pattern for doing this, like ...

```javascript
import SomeModule from './someModule.js';

// snip

if (Module.isEnabled(SomeModule) && SomeModule.options.someOption.value) {
```

Reply:
https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/4919/files#r233343392

Reply:
Made this change and got rid of getOptionValue in the latest commits